package.json

```diff
@@ -272,7 +272,7 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-react-optimize": "^1.0.1",
     "babel-preset-stage-0": "^6.24.1",
-    "brave-ui": "^0.34.7",
+    "brave-ui": "^0.36.0",
```

Comment:
`package` and `package-lock` changes will go away on rebase once https://github.com/brave/brave-core/pull/1028 is merged